README.md

```diff
@@ -7,27 +7,34 @@
 [![Slack](https://slack.buildpacks.io/badge.svg)](https://slack.buildpacks.io/)
 
 `pack` makes it easy for...
-- **App Developers** to use buildpacks to convert code into runnable images.
-- **Buildpack Authors** to develop and package buildpacks for distribution.
-- **Operators** to package buildpacks for distribution and maintain applications.
-
-`pack` is a CLI implementation of the [Platform Interface Specification][platform-spec] for [Cloud Native Buildpacks][buildpacks.io].
+- [**App Developers**][app-dev] to use buildpacks to convert code into runnable images.
+- [**Buildpack Authors**][bp-author] to develop and package buildpacks for distribution.
+- [**Operators**][operator] to package buildpacks for distribution and maintain applications.
 
 ## Usage
 
-- [Installing][install-pack]
-- [Getting Started][getting-started]
+<img src="resources/pack-build.gif" width="600px" />
+
+## Getting Started
```

Comment:
Could we also link to buildpacks Slack?

Reply:
I like that idea! If we're ok, I'd like a new PR created for additional changes. I'm sure there's more we'd like to add but don't want to grow the scope of this indefinitely. 

Do you want to do the honors of that PR? I don't mind if you don't have the availability (or interest :D).